docs/topic_guides/beamline_configuration.rst

```diff
@@ -0,0 +1,128 @@
+###################
+Configuration Files
+###################
+
+.. contents:: Table of Contents
+    :depth: 3
+
+Motivation
+----------
+
+Haven's goal is to provide **support for all of the spectroscopy
+beamlines**. However, each beamline is different, and these
+differences are managed by a **set of configuration** files, similar
+to the .ini files used in the old LabView solution. To keep the
+complexity of these configuration files manageable, Haven gets much of
+the needed information from the IOCs directly.
+
+Haven/Firefly should always load without a specific configuration
+file, but will probably not do anything useful.
+
+Checking Configuration
+----------------------
+
+If Haven is installed with pip, the command ``haven_config`` can be
+used to read configuration variables as they will be seen by Haven:
+
+.. code:: bash
+
+	  $ haven_config beamline
+	  {'is_connected': False, 'name': 'SPC Beamline (sector unknown)'}
+	  $ haven_config beamline.is_connected
+	  False
+
+
+Configuration File Priority
+---------------------------
+
+There are several sources of configuration files, described in detail
+below. They are loaded in the following order, with lower numbers
+taking precedence over higher numbers.
+
+Unless there's a good reason to do otherwise, **most beamline
+configuration belongs in ~/bluesky/iconfig.toml**.
+
+1. Files listed in the ``$HAVEN_CONFIG_FILES``
+2. ``~/bluesky/instrument/iconfig.toml`` (for backwards compatibility)
+3. ``~/bluesky/iconfig.toml``
+4. ``iconfig_default.toml`` packaged with Haven
+
```

Comment:
Sure thing. I added the following below that list of file locations:

```toml
For example, to enable support for our Universal Robotics robot
*Austin* to 25-ID-C, open the file ``~/bluesky/iconfig.toml`` and add
the following:

.. code:: toml
   
   [robot.Austin]
   prefix = "25idAustin"

.. note::

   The prevent accidental changes, the bluesky configuration files may
   not be writable by the user accounts at the beamline. For example,
   at 25-ID, the user account does not have permission to write to
   ``~/bluesky/iconfig.toml`` so **changes must be made as the staff
   account**.
```

Since you approved, I will merge now. Let me know if there's anything else you think should be added.